src/libxtp/mmregion.cc

```diff
@@ -37,7 +37,7 @@ std::string StaticRegion::identify() const {
 }
 
 template <class T>
-void MMRegion<T>::WritePDB(csg::PDBWriter& writer) const {
+void MMRegion<T>::WritePDB(csg::PDBWriter<csg::CSG_Topology>& writer) const {
```

Comment:
is there any other writer than CSG_Topology?

Reply:
Not right now, but I went through the trouble of making it flexible prior to the changes you made in xtp. At this point I could leave it or revert it, Having already done all the work I'm a little hesitant to take the time to revert it unless we are struggling to keep our compile times down.